config/routes.rb

```diff
@@ -195,6 +195,8 @@
               resources :logs, only: [:index]
               scope module: :answer do
                 resources :answers, only: [:show] do
+                  patch :create_programming_files, on: :member
```

Comment:
Any reason why patch was used? Would prefer we just do the REST-ful conventions - ur will be `/answers/:id/programming_files`: post for create and delete to destroy files.

Reply:
Stemming from https://github.com/Coursemology/coursemology2/pull/2436#discussion_r159356180, perhaps the URL will be `courses/:course_id/assessments/:assessment_id/submissions/:submission_id/answers/:answer_id/programming/files/:id` or something along those lines.

Reply:
Wouldn't this be to a `Course::Assessment::Submission::Answer::Programming::FilesController` instead then? (instead of the `Course::Assessment::Submission::Answer::Programming::Controller`)

Reply:
Yup this is ok also.